beacon-chain/sync/initial-sync/blocks_fetcher.go

```diff
@@ -334,6 +532,30 @@ func (f *blocksFetcher) requestBlocks(
 	return prysmsync.SendBeaconBlocksByRangeRequest(ctx, f.chain, f.p2p, pid, req, nil)
 }
 
+func (f *blocksFetcher) requestBlobs(ctx context.Context, req *p2ppb.BlobSidecarsByRangeRequest, pid peer.ID) ([]*p2ppb.BlobSidecar, error) {
+	if ctx.Err() != nil {
+		return nil, ctx.Err()
+	}
+	l := f.peerLock(pid)
+	l.Lock()
+	log.WithFields(logrus.Fields{
+		"peer":     pid,
+		"start":    req.StartSlot,
+		"count":    req.Count,
+		"capacity": f.rateLimiter.Remaining(pid.String()),
+		"score":    f.p2p.Peers().Scorers().BlockProviderScorer().FormatScorePretty(pid),
+	}).Debug("Requesting blobs")
+	if f.rateLimiter.Remaining(pid.String()) < int64(req.Count) {
```

Comment:
The rate limiter here is only for blocks, do we want to use it for blobs too ?

Reply:
My thinking was that since we want to request blocks and blobs from the same peer, we should combine these limits, since the requests stack on each other.

Reply:
Note from conversation with Nishant: we should do some napkin math on the relative block vs blob sizes so that we can apply the right rate limit here. We have tentative agreement on the idea that overriding the block rate limit for blobs accomplishes the goal of reducing block requests in proportion to the fan-out blob requests.